src/middlewared/middlewared/etc_files/docker.py

```diff
@@ -23,8 +23,8 @@ def nvidia_configuration(middleware):
 
 
 def gpu_configuration(middleware):
-    available_gpu = middleware.call_sync('device.get_info', 'GPU')
-    if available_gpu['vendor'] == 'NVIDIA':
+    available_gpus = middleware.call_sync('device.get_info', 'GPU')
+    if any(gpu['vendor'] == 'NVIDIA' for gpu in available_gpus):
```

Comment:
Curious why no Intel here?

Reply:
Because this patch is only required for `nvidia` and docker, for intel gpus, there is no other configuration required on docker end.